Database/Corrections/sodQuestFixes.lua

```diff
@@ -3164,10 +3164,70 @@ function SeasonOfDiscovery:LoadQuests()
             [questKeys.questLevel] = 15,
             [questKeys.requiredRaces] = raceIDs.NONE,
             [questKeys.requiredClasses] = classIDs.SHAMAN,
-            [questKeys.objectivesText] = {"Kill Rot Hide Mystics until a Rot Totem drops, use it to spawn a level 15 elite. Kill it to receieve the rune."},
+            [questKeys.objectivesText] = {"Kill Rot Hide Mystics until a Rot Totem drops, use it to spawn a level 15 elite. Kill it to receive the rune."},
             [questKeys.requiredSpell] = -410107,
             [questKeys.zoneOrSort] = sortKeys.SHAMAN,
         },
+        [90209] = {
+            [questKeys.name] = "Saber Slash",
+            [questKeys.startedBy] = {nil,{414624}},
+            [questKeys.finishedBy] = nil,
+            [questKeys.requiredLevel] = 1,
+            [questKeys.questLevel] = 10,
+            [questKeys.requiredRaces] = raceIDs.NONE,
+            [questKeys.requiredClasses] = classIDs.ROGUE,
+            [questKeys.objectivesText] = {"Loot the Lighthouse Stash under the right of the tree to receive the rune."},
```

Comment:
```suggestion
            [questKeys.objectivesText] = {"Loot the Lighthouse Stash between the roots of the tree to receive the rune."},
```

Reply:
Fixed in: https://github.com/Questie/Questie/pull/5609/commits/82f0883c19e082887899e9a1192485228c484edf